File: fstmerge/examples/Stack/Top/Stack.java
import java.util.LinkedList;

public class Stack<T> {
	private LinkedList<T> items = new LinkedList<T>();
	public Stack() {
		int j = 0;
		int i = 0;
	}
	public void push(T item) {
		items.addFirst(item);
	}
	public T top() {
		return items.getFirst();
	}
	public T pop() {
		int j = 0;
		if(items.size() > 0)
			return items.removeFirst();
		else
			return null;
	}
}

File: fstmerge/examples/Stack_python/Top/stack.py
class Stack:
	items = list()

	def push(self, item):
		self.items.append(item)

	def top(self):
		return self.items[-1]

	def pop(self):
		if len(self.items) > 0:
			return self.items.pop()
		else:
			return None